tests/serialization/triangulation_01.output

```diff
@@ -1,34 +1,34 @@
 
-DEAL::0 0 0 0 0 3 0 2 1 0
-0 1 0 0 1 0 0 0 2 0 0 0 -1 -1 -1 -1 1 0 0 1 0 0 1 0 -1 0 0 0 0 0 1 0 0 0 2 0 1 1 0 0 0 0 0 0 1 1 1 0 0 0 1 0 0 0 0 1 0 4294967295 0 0 1 0 0 1 0 0 0 0 0 2
-1 2 0 0 0 2 0 0 4 0 -1 -1 1 1 1 0 -1 -1 2 0 0 0 2 0 0 0 1 0 0 0 2 0 2 0 2 2 2 1 2 0 0 2 0 0 2 1 1 2 0 0 2 0 0 0 2 0 4294967295 4294967295 1 0 1 2 0 0 0 0 2
-2 4 0 1 0 0 0 4 0 0 0 0 8 0 -1 -1 2 1 2 0 2 2 2 1 2 3 2 2 -1 -1 4 0 1 0 0 0 4 0 4 0 0 0 2 0 0 1 0 4 0 2 0 3 2 3 2 2 2 4 2 4 1 4 0 -1 -1 -1 -1 0 0 4 1 1 1 1 4 1 0 0 0 4 0 2 0 0 0 4 0 4294967295 4294967295 4294967295 4294967295 3 0 1 4 0 3 0 0 0 2 0 0 -1 0 0 5 0 0 0 0 0 1 0 0 0.00000000000000000e+00 1 1.00000000000000000e+00 1 5.00000000000000000e-01 1 2.50000000000000000e-01 1 7.50000000000000000e-01 5 1 1 1 1 1 0 0 0 3 7 3 0 1 2 4 4 3 0 0 0 4 0 0 0 25 1 0
-3 2 0 0 0 0 0 1 1 0 0 28 1 0
+DEAL::0 0 0 0 0 3 0 0 0 3 1 0
```

Comment:
The output is different because boost serializes a `std::unique_ptr` differently from how it serializes a raw pointer. I asked @bangerth about this issue in #2934 and he seemed to think that comparing the output is redundant anyway, since these tests do their own internal checks that the triangulation obtained by de-serializing the output is the same as the original one.